ui/src/app/pages/workspace/workspace-edit.tsx

```diff
@@ -81,6 +85,20 @@ export const styles = reactStyles({
     flex: '1 1 0',
     marginLeft: '1rem'
   },
+  freeCreditsBalanceClickable: {
+    display: 'inline-block',
+    color: colors.accent,
+    padding: '0.25rem 0 0 1rem'
+  },
+  freeCreditsBalanceOverlay: {
+    height: '44px',
+    width: '150px',
+    color: '#262262',
+    fontFamily: 'Montserrat',
+    fontSize: '12px',
```

Comment:
Nit: I think our convention for this is to just put `fontSize: 12`

Reply:
I'll update.  A quick search shows we're very inconsistent here.  This one's my favorite: `fontSize: '.54167rem',`